docs/using-hive.md

```diff
@@ -515,15 +512,15 @@ To use this feature:
          name: hive
        spec:
          managedDomains:
-         - hive.example.com
-         externalDNS:
-           gcp:
+         - gcp:
              credentialsSecretRef:
                name: gcp-creds
+           domains:
+           - hive.example.com
 
-  1. Specify which domains Hive is allowed to manage by adding them to the `managedDomains` list. When specifying `managedDNS: true` in a ClusterDeployment, the ClusterDeployment's baseDomain must be a direct child of one of these domains, otherwise the ClusterDeployment creation will result in a validation error. The baseDomain must also be unique to that cluster and must not be used in any other ClusterDeployment, including on separate Hive instances.
+  1. Specify which domains Hive is allowed to manage by adding them to the `.spec.managedDomains[].domains` list. When specifying `managedDNS: true` in a ClusterDeployment, the ClusterDeployment's baseDomain must be a direct child of one of these domains, otherwise the ClusterDeployment creation will result in a validation error. The baseDomain must also be unique to that cluster and must not be used in any other ClusterDeployment, including on separate Hive instances.
```

Comment:
Can you elaborate on why this has the trailing '.domains' suffix? I would expect just .spec.managedDomains[] here.

Reply:
That is the field that contains the list of domains. HiveConfig looks like the following.
```
spec:
  managedDomains:
  - aws:
      ...
    domains:
    - example-domain.com
```

Reply:
I misread something then, thanks.